checkin.py

```diff
@@ -10,9 +11,15 @@
     users = [line.strip() for line in f.readlines()]
 
 # 获取当前日期和前两天日期
-today = datetime.date.today()
+beijing_timezone = pytz.timezone('Asia/Shanghai')
```

Comment:
Can you provide a direct result you get from your console for me. It will be easier to know the effect.

Reply:
```shell
➜   ./checkin.py
今天是：2023-05-26, 检查过去两天打卡情况: 2023-05-25 2023-05-24
当前6位同学参与打卡，1位老板缺卡，缺卡率：16.67%。请发2元红包5份，再发一个2元红包给我。谢谢老板！
```

Reply:
Cool.